rfcs/expected_fail_message_tag_for_subtests.md

```diff
@@ -0,0 +1,71 @@
+# RFC 165: Support expected-fail-message in test metadata for subtests
+
+## Summary
+
+Add "expected-fail-message" in test metadata so that we can track how
+a subtest fails.
```

Comment:
Do we also need to track harness error messages?

Reply:
Asked this in crbug.com/1481730.

Reply:
I think the answer should probably be yes. When there's a harness error, the exception message is often enough to figure out the problem, so having the importer write that to the repo by default would be convenient, and is the current behavior with -expected.txt.

Reply:
Include the harness error message in actual output would be convenient for devs to know what is wrong, but do we also want to use that to prevent regression? If that is the case, we would need another key "expected-harness-error-message"? Is there a way we could consolidate the two into one?

If we do not need to track regression, we can still include the error message in actual output, but convert harness error into a test level expected error when converting baselines to ini files.

Reply:
I'm not against adding a way to test against "expected" harness errors, but this is certainly a much less common case. Typically/ideally these are truly harness errors, not something failing in a way we'd like to continue testing for. I could be proven wrong with examples. But if I'm right, this can be lower priority.

Reply:
I discussed with foolip@ offline, and he agreed this can be lower priority, so resolve it now.